### ISSUE TITLE
Bug: Server side component fails on stale sessions/tokens

### ISSUE THREAD

**Issue by yelodevopsi** [varianter/vibes#148]
When refreshing `https://vibes-frontend-dev.azurewebsites.net/bemanning?search=&filter=` 

From the logs:

```
2023-10-18T12:25:06.984607532Z SyntaxError: Unexpected end of JSON input
2023-10-18T12:25:06.984681131Z     at JSON.parse (<anonymous>)
2023-10-18T12:25:06.984686531Z     at parseJSONFromBytes (node:internal/deps/undici/undici:4553:19)
2023-10-18T12:25:06.984689531Z     at successSteps (node:internal/deps/undici/undici:4527:27)
2023-10-18T12:25:06.984692131Z     at fullyReadBody (node:internal/deps/undici/undici:1307:9)
2023-10-18T12:25:06.984694631Z     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-10-18T12:25:06.984697331Z     at async specConsumeBody (node:internal/deps/undici/undici:4536:7)
2023-10-18T12:25:06.984699931Z     at async i (/app/build/server/app/bemanning/page.js:1:7775)
2023-10-18T12:25:06.984712031Z     at async u (/app/build/server/app/bemanning/page.js:1:5957)
2023-10-18T12:25:06.993022282Z SyntaxError: Unexpected end of JSON input
2023-10-18T12:25:06.993037281Z     at JSON.parse (<anonymous>)
2023-10-18T12:25:06.993041181Z     at parseJSONFromBytes (node:internal/deps/undici/undici:4553:19)
2023-10-18T12:25:06.993044681Z     at successSteps (node:internal/deps/undici/undici:4527:27)
2023-10-18T12:25:06.993047681Z     at fullyReadBody (node:internal/deps/undici/undici:1307:9)
2023-10-18T12:25:06.993051181Z     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-10-18T12:25:06.993054281Z     at async specConsumeBody (node:internal/deps/undici/undici:4536:7)
2023-10-18T12:25:06.993057381Z     at async i (/app/build/server/app/bemanning/page.js:1:7775)
2023-10-18T12:25:06.993060381Z     at async u (/app/build/server/app/bemanning/page.js:1:6947)
2023-10-18T12:25:07.002953922Z [Error: An error occurred in the Server Components render. The specific message is omitted in production builds to avoid leaking sensitive details. A digest property is included on this error instance which may provide additional details about the nature of the error.] {
2023-10-18T12:25:07.002968722Z   digest: '3052080722'
2023-10-18T12:25:07.002972222Z }
2023-10-18T12:25:07.005150909Z [Error: An error occurred in the Server Components render. The specific message is omitted in production builds to avoid leaking sensitive details. A digest property is included on this error instance which may provide additional details about the nature of the error.] {
2023-10-18T12:25:07.005161209Z   digest: '3425567632'
2023-10-18T12:25:07.005164209Z }
```

**Comment by yelodevopsi** [comment]
Deleted `__Secure-next-auth.session-token.1`
Now redirects to `https://vibes-frontend-dev.azurewebsites.net/api/auth/signin` correctly

**Comment by yelodevopsi** [comment]
Most likely because we set maxAge to 30 days and not in sync with the actual `exp` timestamp on the token.

Read mostly 1h 30min but setting to 1h to ensure its refreshed anyways

**Comment by yelodevopsi** [comment]
Solved in #149
